chpl/chpl-api/e2e/env/sample.postman_environment.json

```diff
@@ -11,6 +11,26 @@
             "key": "apiKey",
             "value": "enterKeyHere",
             "enabled": true
+        },
+		{
+            "key": "authurl",
+            "value": "https://chpl.ahrqdev.org/rest/auth/authenticate",
```

Comment:
We may want to have this url as generic url name

Reply:
Pushed a commit for this